src/generic/grid.cpp

```diff
@@ -58,6 +61,17 @@ const char wxGridNameStr[] = "grid";
 // Required for wxIs... functions
 #include <ctype.h>
 
+// Drawing transparent selection can only be done via a graphics context,
+// or a DC that has a native support for drawing with alpha, like standard
+// DCs on OSX and GTK3.
```

Comment:
Sorry, it's a pity that you've already done it, but I think there might have been a misunderstanding in one of the previous discussions: I never wanted to imply that you should support using `wxDC` when `wxUSE_G_C==0` in these ports. First of all, I don't even this it can be disabled in them and, second, even if it could, it's definitely not worth it because nobdoy would do it in practice.

So I'd just remove the port checks and use only `#if wxUSE_G_C` (although you might want to `#define wxHAS_TRANSPARENT_SELECTION` in this case and check for this one below to make these checks more readable).